source/partials/next_release/_release.md.erb

```diff
@@ -1,37 +1,59 @@
-<h4>some feature</h4>
+<h4> New Pipeline Creation Wizard </h4>
 
-<h4>Versions of Java this release of GoCD works with</h4>
-<ul>
-  <li>Java 8 (Deprecated)</li>
-  <li>Java 10 (Deprecated)</li>
-  <li>Java 11</li>
-  <li>Java 12</li>
-</ul>
+We have introduced a new simplified pipeline creation wizard towards improving the user experience.
 
-<h4>API Improvements</h4>
+Feedback/ideas are welcome, please open an issue [here](https://github.com/gocd/gocd/issues) to add your comments.
 
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/19.8.0/new-pipeline-wizard.gif', alt: 'New Pipeline Creation Wizard' %>
+</figure>
 
-<h4>Other Improvements</h4>
+<h4>Introduce support for latest API version</h4>
 
+GoCD API's are versioned, accessing an API enpoint required providing a version number as part of the `Accept: application/vnd.go.cd.v3+json` header.
 
-<h4>Bug fixes</h4>
+GoCD now allows accessing the latest version of an API, this can be done by omitting the version number in the request header `Accept: application/vnd.go.cd+json`. More information can be found [here](https://github.com/gocd/gocd/issues/6620).
```

Comment:
GoCD now allows accessing the latest version of an API. This can be done by omitting the version number in the request header `Accept: application/vnd.go.cd+json`. More information can be found [here](https://github.com/gocd/gocd/issues/6620).

Reply:
Updated to:
`GoCD now allows accessing the latest version of an API by omitting the version number in the request header Accept: application/vnd.go.cd+json. More information can be found here.`